src/ImageSharp/Formats/Jpeg/PdfJsPort/Components/PdfJsScanDecoder.cs

```diff
@@ -432,66 +412,86 @@ private void DecodeMcuACSuccessive(ref PdfJsHuffmanTable acHuffmanTable, PdfJsFr
         }
 
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
-        private int ReadBit(DoubleBufferedStreamReader stream)
+        private bool TryReadBit(DoubleBufferedStreamReader stream, out int bits)
```

Comment:
Why this change?

Exchanging an `int` + a `bool` through the stack is slower than doing the same using only an `int`, and we are on a hot path here AFAIK.